tdd_buckets.py

```diff
@@ -0,0 +1,62 @@
+ranges = []
+first_number = 0
+last_number = 0
+range_count = 0
+
+
+def check_ranges_on_array(array):
+    global range_count
+    for x in array:
+        range_count += 1
+        validate_range_value(x)
+    write_range(first_number, last_number, range_count)
+    return send_result(ranges)
+
+
+def validate_range_value(value):
+    global first_number
```

Comment:
Consider using a class, or passing around a dictionary - instead of using global variables